build-tools/enumification-helpers/generate-const-mapping.cs

```diff
@@ -120,7 +120,7 @@ void Run (string [] args)
 						val,
 						string.IsNullOrEmpty (forceApiLevel) ? api : forceApiLevel);
 				}
-				} catch (Exception ex) {
+				} catch (Exception) {
```

Comment:
Wouldn't we want the exception value to be printed?

Reply:
That's a good point I removed the var ex because it was not being used and we were getting a warning, but the right fix would be to include the error on the print. thanks for catching that!